client/blocks/login/social.jsx

```diff
@@ -88,7 +88,7 @@ class SocialLoginForm extends Component {
 			() => {
```

Comment:
Sending `redirectTo` to the parent component is not needed as the `loginSocialUser` redux action will actually handle it in the redux state.
It will send `redirect_to` as part of the login request and update the `redirect_to` reducer on server's response.